high_card.rb

```diff
@@ -1,5 +1,37 @@
 # Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
-suits = [ "hearts", "spades", "clubs", "diamonds" ]
 
+ranks = [ 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K", "A" ]
+suits = [ "hearts", "spades", "clubs", "diamonds" ]
 players = []
+deck = []
+deck_shuffle = []
+shuffled_fiftytwo = []
+deck2 = []
+name2 = []
+
+# prompts player
+print "What is your name?"
+name = gets.chomp
+   name2 = name.capitalize
+name.capitalize!
+print "Hello, " + name + ". Welcome to war!"
+print "Play War!"
```

Comment:
So it's great you were able to get a player name, but this isn't quite what the directions asked for. The game shouldn't run until 'play' was typed in by the user.